phoenix-shared/src/commonMain/kotlin/fr.acinq.phoenix/db/SqliteAppDb.kt

```diff
@@ -63,16 +62,16 @@ class SqliteAppDb(driver: SqlDriver) {
         return getWalletContextOrNull(version).second
     }
 
-    suspend fun getWalletContextOrNull(version: WalletContext.Version): Pair<Instant, WalletContext.V0?> =
+    suspend fun getWalletContextOrNull(version: WalletContext.Version): Pair<Long, WalletContext.V0?> =
         withContext(Dispatchers.Default) {
             paramsQueries.get(version.name, ::mapWalletContext).executeAsOneOrNull()
-        } ?: Instant.DISTANT_PAST to null
+        } ?: Clock.System.now().toEpochMilliseconds() to null
```

Comment:
Previously, the code used `DISTANT_PAST`, and now it uses `now()`. Does this matter ?

Reply:
Thanks! Fixed in 2ead172